documentation/docs/14-configuration.md

```diff
@@ -68,6 +68,10 @@ const config = {
 		},
 		target: null,
 		trailingSlash: 'never',
+		version: {
+			name: Date.now(),
+			pollInterval: 60_000,
```

Comment:
After discussion with other maintainers, this should probably default to `0`, which would mean 'no polling'